tg/tdlib.py

```diff
@@ -1,8 +1,25 @@
+from enum import Enum
 from typing import Any, Dict, List
 
 from telegram.client import AsyncResult, Telegram
 
 
+class Action(Enum):
+    chatActionTyping = "typing"
+    chatActionCancel = "cancel"
+    chatActionRecordingVideo = "recording video"
+    chatActionUploadingVideo = "uploading video"
+    chatActionRecordingVoiceNote = "recording voice note"
+    chatActionUploadingVoiceNote = "uploading voice note"
```

Comment:
it is too verbose labels. telegram desktop shows only `recording voice`, so 3rd word `note` could be omitted.  
<img width="287" alt="image" src="https://user-images.githubusercontent.com/9078927/85929378-e8f5b280-b8bc-11ea-9f0d-7d8f27e6dc9b.png">